### PR TITLE
XFA - Default fill color for rectangle is transparent

### PR DESCRIPTION
**Pull request by calixteman**
The issue is visible in pdf in https://github.com/mozilla/pdf.js/issues/13213 (some black rectangles are displayed).

### CODE REVIEW

File: src/core/xfa/template.js [hunk 1]
```diff
@@ -2462,14 +2462,17 @@ class Fill extends XFAObject {
...
   [$toStyle]() {
     const parent = this[$getParent]();
+    const style = Object.create(null);
+
     let propName = "color";
     if (parent instanceof Border) {
       propName = "background";
     }
-    if (parent instanceof Rectangle) {
+    if (parent instanceof Rectangle || parent instanceof Arc) {
```
Comment (marco-c) [inline src/core/xfa/template.js:2471]: Is Arc needed for the same PDF? Or is it a defensive change?

**Review by marco-c — APPROVED** [review]

File: src/core/xfa/template.js (hunk 1 above)
Comment (calixteman) [reply src/core/xfa/template.js:2471]: yep it is.

**Comment by timvandermeij** [comment]
@calixteman The Windows bot seems to be unavailable since yesterday, and it's also missing here. Could you restart it?

**Comment by calixteman** [comment]
> @calixteman The Windows bot seems to be unavailable since yesterday, and it's also missing here. Could you restart it?

Done.
The vm has rebooted by itself, I don't know why... anyway I just restarted the bot.

**Pull request merged**